test/kwtool_web/controllers/api/v1/error_handler_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule KwtoolWeb.Api.V1.ErrorHandlerTest do
+  use KwtoolWeb.ConnCase, async: true
+
+  alias KwtoolWeb.Api.V1.ErrorHandler
+
+  describe "auth_error/2" do
+    test "renders 401 error page", %{conn: conn} do
```

Comment:
```suggestion
    test "renders 401 error response", %{conn: conn} do
```

Reply:
Updated in https://github.com/byhbt/kwtool/pull/45/commits/5c56d403ac73dd8c41a089f0f6397895627d3386